rust/ql/lib/codeql/rust/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -499,6 +522,14 @@ class ReturnExprTree extends PostOrderTree instanceof ReturnExpr {
   }
 }
 
+class StaticTree extends StandardPostOrderTree, Static {
```

Comment:
I think a `static` block should be a control flow scope, as far as I know they are not part of the normal control flow of a function.